bridge/bridgeapp/views.py

```diff
@@ -73,42 +52,34 @@ def post(self, request, category_id, category_slug):
                 for cat_id in cat_ids:
                     category = Category.objects.get(id=cat_id)
                     thread.categories.add(category)
-                id, slug = cat_ids[0], slugify(Category.objects.get(id=cat_ids[0]).type)
+        delete_commented_out_code
```

Comment:
What does line 55 do? It's not a comment, nor code that does anything.

Reply:
I'm not sure how that got in there